src/main/webapp/index-script.js

```diff
@@ -58,7 +59,23 @@ function loadMainButtons() {
   });
   
   logInButton.addEventListener("click", () => {
-    window.location.href = "/login"
+    window.location.href="/login" 
+  }); 
+}
+
+// Chooses whether to display 'Login' or 'Logout' button.
+function toggleLoginLogout(){
```

Comment:
### nit
Good function name, perhaps could be more concise, although I don't know how.